db-tests/src/test/java/gov/nasa/jpl/aerie/database/DatabaseTests.java

```diff
@@ -252,12 +262,63 @@ void clearSimulations() throws SQLException {
     }
   }
 
+  int insertDataset() throws SQLException {
+    try (final var statement = connection.createStatement()) {
+      final var res = statement
+          .executeQuery(
+              """
+                  INSERT INTO dataset
+                  DEFAULT VALUES
+                  RETURNING id;"""
+          );
+      res.next();
+      return res.getInt("id");
+    }
+  }
+
+  void clearDatasets() throws SQLException {
+    try (final var statement = connection.createStatement()) {
+      statement
+          .executeUpdate(
+              """
+                  TRUNCATE dataset CASCADE;"""
+          );
+    }
+  }
+
+  PlanDatasetId insertPlanDataset(final int planId) throws SQLException {
+    try (final var statement = connection.createStatement()) {
+      final var res = statement
+          .executeQuery(
+              """
+                  INSERT INTO plan_dataset (plan_id, offset_from_plan_start)
+                  VALUES ('%s', '0')
+                  RETURNING plan_id, dataset_id;"""
+                  .formatted(planId)
+          );
+      res.next();
+      return new PlanDatasetId(res.getInt("plan_id"), res.getInt("dataset_id"));
+    }
+  }
+
+  void clearPlanDatasets() throws SQLException {
+    try (final var statement = connection.createStatement()) {
+      statement
+          .executeUpdate(
+              """
+                  TRUNCATE plan_dataset CASCADE;"""
+          );
+    }
+  }
+
   int fileId;
   int missionModelId;
   int planId;
   int activityId;
   int simulationTemplateId;
   int simulationId;
+  int datasetId;
+  PlanDatasetId planDatasetId;
```

Comment:
The datasetID should be included in the `PlanDatasetId` record, which is not just holding a plan ID (see comment about renaming the record). Also, see comment on the `beforeEach()` method below.

Reply:
Agreed. I'm going to leave the dataset insert method in there for possible future use cases, but just remove the call and assignment